<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>179</width>
      <height>159</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="timer_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mine_layout">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcdNumber_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menua">
    <property name="title">
     <string>游戏</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>新游戏</string>
     </property>
     <addaction name="actionfirst"/>
     <addaction name="actionsecond"/>
     <addaction name="actionthrid"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="separator"/>
    <addaction name="actionrecord"/>
   </widget>
   <addaction name="menua"/>
  </widget>
  <action name="actionfirst">
   <property name="text">
    <string>初级</string>
   </property>
  </action>
  <action name="actionsecond">
   <property name="text">
    <string>中级</string>
   </property>
  </action>
  <action name="actionthrid">
   <property name="text">
    <string>高级</string>
   </property>
  </action>
  <action name="actionself_define">
   <property name="text">
    <string>自定义</string>
   </property>
  </action>
  <action name="actionrecord">
   <property name="text">
    <string>统计信息</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
